modules/microshift-install-bootc-workflow.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-about-rhel-image-mode
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-install-rhel-image-mode-conc_{context}"]
+= The image mode for {op-system-base} with {microshift-short} workflow
+
+To use image mode for {op-system-base}, ensure that the following resources are available:
+
+* A {op-system-base} {op-system-version} host with an active Red{nbap}Hat subscription for building {microshift-short} bootc images.
```

Comment:
There are two problems with this sentence:
- `{nbap}` is not rendered correctly
- The RHEL version is rendered to 9.6 - it should be 9.4 in 4.18.